pax/plugins/io/WaveformSimulator.py

```diff
@@ -118,9 +118,25 @@ def s2(self, electrons, g4_id=-1, t=0., x=0., y=0., z=0.):
             x = r * np.cos(phi)
             y = r * np.sin(phi)
 
-        return self.simulator.queue_signal(photon_times, x, y,
+        arriving_photon_times_top,  arriving_photon_times_bottom = self.simulator.queue_signal(photon_times, x, y,
                                            # Generate S2 hitpattern "at the anode": cue for simulator to use S2 LCE map
                                            z=-self.config['gate_to_anode_distance'])
+        peak_top_fraction = float(len(arriving_photon_times_top)) / float(
+            len(arriving_photon_times_bottom) + len(arriving_photon_times_top) )
+        self.store_true_peak('s2',
+                                          g4_id,
+                                          t,
+                                          x,
+                                          y,
+                                          z,
+                                          np.concatenate((
+                                                                   arriving_photon_times_top,
+                                                                   arriving_photon_times_bottom
+                                                                   )
+                                                                  ),
+                                          electron_times,
+                                          )
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->